.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,28 +1,18 @@
 ---
-name: 🧠  Feature request
-about: Suggest an idea for this project
-
+name: 🧠  Feature Request
+about: Suggest an idea for improving Self-Hosted Sentry
 ---
 
-<!--
```

Comment:
Why remove this?

Reply:
Because our triage and public roadmap processes are undefined right now. We could maybe bring this back on the other side of https://github.com/getsentry/open-source-meta/issues/3 and https://github.com/getsentry/open-source-meta/issues/4.